ui/pages/confirmations/components/confirm/header/header-info.tsx

```diff
@@ -90,6 +98,25 @@ const HeaderInfo = () => {
             }}
           />
         </Tooltip>
+        <Box
+          backgroundColor={
+            showAdvancedDetails
+              ? BackgroundColor.infoMuted
+              : BackgroundColor.transparent
+          }
+          borderRadius={BorderRadius.MD}
+        >
+          <ButtonIcon
```

Comment:
Is this currently shown in all confirmations?

Do we want to limit that to those that use it so just contract interaction for now?